minecraft_servers/timenation/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "server_name": "timenation",
+  "nice_name": "TimeNation",
+  "direct_ip": "timenation.net",
+  "server_wildcards": [
+    "%.timenation.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://www.timenation.net",
```

Comment:
Is the maintenance temporary? If it is planned for a longer time period, please remove the link and create a new PR as soon as the website is released.

Reply:
No its not. We will Release the Website shortly.